packages/discovery-react-components/src/components/SearchInput/__tests__/SearchInput.test.tsx

```diff
@@ -188,11 +188,7 @@ describe('<SearchInput />', () => {
           });
 
           test('no autocompletions are displayed', () => {
-            waitForElementToBeRemoved(() =>
-              queryByTestId(container, 'completions-dropdown-test-id')
-            ).then(() => {
-              expectNoCompletionsDropdown(container);
-            });
+            expectNoCompletionsDropdown(container);
```

Comment:
I don't believe this test was ever working, since it didn't catch issue that `showAutocomplete` was defaulting to `true` (see previous comment). To actually work, this would have needed to be an `async` function and return the result of `waitForElementToBeRemoved()`.

But the waitFor wasn't even needed in this case.